parlai/tasks/dbll_babi/agents.py

```diff
@@ -55,113 +55,8 @@ def __init__(self, opt, shared=None):
         opt['datafile'] = _path(os.path.join('babi', 'babi1'), params, opt)
```

Comment:
so dbll_babi hasnt changed?

Reply:
No, I reverted the changes I made before